runtime/connections.go

```diff
@@ -198,26 +200,20 @@ func (r *Runtime) ConnectorConfig(ctx context.Context, instanceID, name string)
 		}
 	}
 
-	// Search for connector definition in rill.yaml
+	// Search for connector definitions from YAML files
 	vars := inst.ResolveVariables(true)
```

Comment:
This will make the template resolution case insensitive. This was okay previously because variables could only refer to `connector....` but now any variable can be referred. 
Though this should still be okay because mostly credentials will be referred.

Reply:
Good point, don't want that! Changed it :)